iceoryx_examples/icecubetray/README.md

```diff
@@ -0,0 +1,127 @@
+# icecubetray
+
+## Introduction
+
+This example demonstrates how access rights can be applied to shared memory segments on Linux-based operating systems.
+It provides a custom RouDi, a radar and a display application.
+
+!!! hint
+    The access right feature is only supported on Linux and QNX
+
+## Expected output
+
+<!-- Add asciinema link here -->
+
+## Code walkthrough
+
+The user _roudi_ does not need root access rights. However, it needs _CAP\_KILL_ capability or similar rights on
+other POSIX operating system. RouDi needs to be able to send a _SIGKILL_ signal to the apps in case RouDi is shutdown.
+
+| Users        | privileged group | unprivileged group | infotainment group |   iceoryx group    |
+|--------------|:----------------:|:------------------:|:------------------:|:------------------:|
+| perception   |        X         |                    |                    |         X          |
+| infotainment |                  |         X          |         X          |         X          |
+| roudi        |                  |                    |                    |         X          |
+| notallowed   |                  |                    |                    |         X          |
+
+### Overview over the Apps and Shared Memory Segments
+
+```
```

Comment:
You're right, I've adapted the picture.

> If you do not have write access and write into such a segment you will encounter a segmentation fault (for instance when Display App would like to write into Privileged Shared).

That's the behaviour that we want avoid in future. Segfaults shall never be the expected behaviour in reliable software. If no write access is given and the user creates a publisher object, the users gets an error message and program execution stops. In case of a subscriber without read access there will be just not data arriving. The subscriber behaviour will be implemented on a separate PR for v2.0